pkg/entrypoint/tlsproxy/cmd.go

```diff
@@ -0,0 +1,80 @@
+package tlsproxy
```

Comment:
note to self: I would like to maintain a cmd/ entrypoint for tlsproxy *as well as* building it into cmd/azure, this will be useful for the vm image build